### PR TITLE
Update to latest from build 10.12.1.0.6 (IFS 10.12.1.0.7)

### PR DESCRIPTION
**Pull request by gregkresge-cornelisnetworks**
Successfully built manually on RHEL9 (bld-node-42)

Repo release and tag will be added after merging to master.

### CODE REVIEW

File: IbaTools/FastFabric/upgrade.exp [hunk 1]
```diff
@@ -124,7 +124,7 @@ File: /usr/lib/opa/tools/upgrade.exp" case_setup case_cleanup {
 			# given driver, its possible that its not on the media
 			# so we can't expect it to be installed
 			# for now assume Stack or Enablement tools are always on media
-			set drivers {{((OFED OPA Stack)|(OFA OPA Stack)|(OFED IBACM)|(OFA IBACM)|(Cornelis HFI Components)|(OPA Tools)|(OFED IP over IB)|(OFA IP over IB)|(OPA FM))}}
+			set drivers {{((OFED OPA Stack)|(OFA OPA Stack)|(OFED IBACM)|(OFA IBACM)|(Intel HFI Components)|(OPA Tools)|(OFED IP over IB)|(OFA IP over IB)|(OPA FM))}}
```
Comment (davidgonzalez-cornelisnetworks) [inline IbaTools/FastFabric/upgrade.exp]: A rebranding change seems to be getting undone here.

File: TestTools/chassis.exp [hunk 2]
```diff
@@ -1426,7 +1426,7 @@ proc check_esm_config { config_file } {
 			error "$tooldir/config_check -c $config_file: failed: $err_str"
 		}
 	} else {
-		skip_case "Unable to pre-check config file, Cornelis OPX FM not installed on this server"
+		skip_case "Unable to pre-check config file, Intel OPA FM not installed on this server"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/chassis.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 3]
```diff
@@ -452,7 +452,7 @@ proc get_switch_brand { ibnode } {
 		if { [ string equal "$deviceid" "0xabc0"] } {
 			if { [ string equal "$vendorid" "0x8086"] } {
 				set brand "OPA"
-				log_message "$ibnode: detected Cornelis OPX device"
+				log_message "$ibnode: detected Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 4]
```diff
@@ -610,16 +610,16 @@ proc get_switch_info { ibnode } {
 			set out [expect_list_with_punchlist "$ibnode" "Unable to query switch" 60 { "FW Version is [0-9.]+" } { "Error"}]
 			regexp {([0-9.]+)} $out fwversion
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 5]
```diff
@@ -610,16 +610,16 @@ proc get_switch_info { ibnode } {
...
 		}
 
         # use verbose option so we have a predictable output to expect
 		if { [ string equal "$brand" "OPA"] } {
 			send_unix_cmd "$toolsdir/opaswquery -t $trimmed_ibnode $hfi_port_args -Q 11"
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 6]
```diff
@@ -631,8 +631,8 @@ proc get_switch_info { ibnode } {
 		if { [ string equal "$brand" "OPA"] } {
 			send_unix_cmd "$toolsdir/opaswquery -t $trimmed_ibnode $hfi_port_args -Q 4"
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 7]
```diff
@@ -738,8 +738,8 @@ proc get_switch_info { ibnode } {
 			}
 			set fanStatus "$fanaStatus/$fanbStatus/$fancStatus/$fandStatus/$faneStatus/$fanfStatus"
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 8]
```diff
@@ -771,8 +771,8 @@ proc get_switch_info { ibnode } {
 			set tempStatus "$tempaStatus/$tempbStatus/$tempcStatus"
 
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 9]
```diff
@@ -809,8 +809,8 @@ proc get_switch_info { ibnode } {
 				set ps2Status "$p3"
 			}
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 10]
```diff
@@ -851,8 +851,8 @@ PS2 Status:$ps2Status
 Temperature status:$tempStatus"
 			}
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 11]
```diff
@@ -889,7 +889,7 @@ proc get_switch_hwvpd { ibnode } {
 			send_unix_cmd "$toolsdir/opaswquery -t $trimmed_ibnode $hfi_port_args -Q 4"
 		} else {
 			append_punchlist "$ibnode" "Not an OPA series device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 12]
```diff
@@ -953,8 +953,8 @@ proc get_switch_ping { ibnode { hfi_port ""} { report "1" } { ignore_pass "0" }
 				check_exit_status 60 0
 				send_unix_cmd "$toolsdir/opaswping -t $trimmed_ibnode $hfi_port_args"
 			} else {
-				append_punchlist "$ibnode" "Not an Cornelis OPX device"
-				fail_test "$ibnode: Not an Cornelis OPX device"
+				append_punchlist "$ibnode" "Not an Intel OPA device"
+				fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 13]
```diff
@@ -1003,8 +1003,8 @@ proc get_switch_fwverify { ibnode } {
 			check_exit_status 60 0
 			send_unix_cmd "$toolsdir/opaswfwverify -t $trimmed_ibnode $hfi_port_args -q"
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 14]
```diff
@@ -1074,8 +1074,8 @@ proc get_switch_capture { ibnode upload_file } {
 			send_unix_cmd "$toolsdir/xedge_capture -v -t $node_guid $hfi_port_args $upload_file"
 			check_exit_status 60 0
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 15]
```diff
@@ -1219,8 +1219,8 @@ File: TestTools/extmng.exp" case_setup case_cleanup {
 				expect_list_with_punchlist "$ibnode" "Unable to update FW" 300 { "opaswfwupdate completed"} { "Error" } 
 			}
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 16]
```diff
@@ -1320,7 +1320,7 @@ File: TestTools/extmng.exp" case_setup case_cleanup {
 				expect_list_with_punchlist "$ibnode" "Unable to reboot switch" 300 { "opaswreset completed"} { "Error" } 
 			}
 		} else {
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 17]
```diff
@@ -1455,10 +1455,10 @@ File: TestTools/extmng.exp" case_setup case_cleanup {
 		set brand [get_switch_brand "$ibnode"]
 		if { [ string equal "$brand" "OPA"] } {
 			# are we going to implement this?
-			puts "Function not yet implemented for Cornelis OPX"
+			puts "Function not yet implemented for Intel OPA"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 18]
```diff
@@ -1455,10 +1455,10 @@ File: TestTools/extmng.exp" case_setup case_cleanup {
...
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

File: TestTools/extmng.exp [hunk 19]
```diff
@@ -1609,8 +1609,8 @@ File: TestTools/extmng.exp" case_setup case_cleanup {
 			}
 
 		} else {
-			append_punchlist "$ibnode" "Not an Cornelis OPX device"
-			fail_test "$ibnode: Not an Cornelis OPX device"
+			append_punchlist "$ibnode" "Not an Intel OPA device"
+			fail_test "$ibnode: Not an Intel OPA device"
```
Comment (davidgonzalez-cornelisnetworks) [inline TestTools/extmng.exp]: Rebranding.

**Review by davidgonzalez-cornelisnetworks — CHANGES_REQUESTED** [review]

**Comment by gregkresge-cornelisnetworks** [comment]
@davidgonzalez-cornelisnetworks Slightly concerned why we are seeing all of these branding reversals. Were these never cherry-picked over to opa-dev/opa10_12 (seeing these "Intel" references in ifs-all)? Or am I forgetting that this was something that was separately, manually done on the public repos? Either way, I'll get these corrected. Good catches.

**Comment by gregkresge-cornelisnetworks** [comment]
Looks like this patch was applied to opa-10_11, but never cherry-picked to opa-dev

commit dd7a846049c439bad536ef3bb5942524dac57cf9
Author: Scott Breyer <scott.breyer@cornelisnetworks.com>
Date:   Thu Oct 28 11:02:45 2021 -0400

    STL-62361 Rebranding for OpenIB Install
    
    Reviewed-by: Gonzalez, David david.gonzalez@cornelisnetworks.com
    Reviewed-by: Kresge, Greg greg.kresge@cornelisnetworks.com
    Signed-off-by: Breyer, Scott scott.breyer@cornelisnetworks.com
    (cherry picked from commit 98513213c5138ac35b9dee9455985a401b21b8ef)

File: IbaTools/FastFabric/load.exp [hunk 20]
```diff
@@ -124,7 +124,7 @@ File: /usr/lib/opa/tools/load.exp" case_setup case_cleanup {
 			# given driver, its possible that its not on the media
 			# so we can't expect it to be installed
 			# for now assume Stack or Enablement tools are always on media
-			set drivers {{((OFED OPA Stack)|(OFA OPA Stack)|(OFED IBACM)|(OFA IBACM)|(Cornelis HFI Components)|(OPA Tools)|(OFED IP over IB)|(OFA IP over IB)|(IFS FM))}}
+			set drivers {{((OFED OPA Stack)|(OFA OPA Stack)|(OFED IBACM)|(OFA IBACM)|(Intel HFI Components)|(OPA Tools)|(OFED IP over IB)|(OFA IP over IB)|(IFS FM))}}
```
Comment (gregkresge-cornelisnetworks) [inline IbaTools/FastFabric/load.exp]: Branding

File: IbaTools/FastFabric/opa_ca_openssl.cnf [hunk 21]
```diff
@@ -1,5 +1,5 @@
 #
-# Cornelis OPX Self-Signed CA OpenSSL example configuration file.
+# Intel OPA Self-Signed CA OpenSSL example configuration file.
```
Comment (gregkresge-cornelisnetworks) [inline IbaTools/FastFabric/opa_ca_openssl.cnf]: Branding

File: IbaTools/FastFabric/opa_comp_openssl.cnf [hunk 22]
```diff
@@ -1,5 +1,5 @@
 #
-# Cornelis OPX FM/FF Component OpenSSL example configuration file.
+# Intel OPA FM/FF Component OpenSSL example configuration file.
```
Comment (gregkresge-cornelisnetworks) [inline IbaTools/FastFabric/opa_comp_openssl.cnf]: Branding

File: IbaTools/FastFabric/upgrade.exp (hunk 1 above)
Comment (gregkresge-cornelisnetworks) [inline IbaTools/FastFabric/upgrade.exp]: Branding

Comment (gregkresge-cornelisnetworks) [reply IbaTools/FastFabric/upgrade.exp]: Corrected, thanks

File: TestTools/chassis.exp (hunk 2 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/chassis.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 3 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 5 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 6 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 7 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 8 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 15 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 16 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 17 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 19 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 18 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

File: TestTools/extmng.exp (hunk 4 above)
Comment (gregkresge-cornelisnetworks) [reply TestTools/extmng.exp]: Corrected, thanks

**Comment by gregkresge-cornelisnetworks** [comment]
Amended commit to not revert rebranding. While I'll get a separate patch staged on opa-dev to address this delta, I think that should cover the misses in this PR.

**Review by davidgonzalez-cornelisnetworks — APPROVED** [review]

**Pull request merged**